src/components/SQForm/SQFormMultiSelect.tsx

```diff
@@ -21,18 +22,16 @@ import {
   getUndefinedValueWarning,
 } from '../../utils/consoleWarnings';
 import type {TooltipProps, SelectProps} from '@material-ui/core';
+import type {ListChildComponentProps} from 'react-window';
 import type {BaseFieldProps, SQFormOption} from '../../types';
 
-export type SQFormMultiSelectProps = BaseFieldProps & {
+export type SQFormMultiSelectProps<TVirtualized = boolean> = BaseFieldProps & {
```

Comment:
Do we need this generic type since it seems like the prop should always be a boolean?

Reply:
Ya, it's a boolean. But I'm using the value of the boolean to determine what type of event will be sent to the onChange handler. It's mainly to keep TS happy. Consumers should never have to supply the generic.